src/Pillar-ExporterCore/PRTarget.class.st

```diff
@@ -8,6 +8,66 @@ Class {
 	#category : #'Pillar-ExporterCore-Base'
 }
 
+{ #category : #'as yet unclassified' }
+PRTarget class >> associateTarget: aFormat [
+	^ self allSubclasses
+		select: [ :each | (each isAbstract not) and: each builderName = aFormat asString ]
+]
+
+{ #category : #clap }
+PRTarget class >> buildClapCommand [
+	"When CLAP issue about ambiguous action with subcommand/positionnal will be solved, replace the next lign by the following one: 
+		self buildClapCommandWithId: self clapId"
```

Comment:
I think the comment should explain WHY we do it. But it should not contain code like that. That is for an issue.

Reply:
So I create an issue in Pillar-markup, giving all concerned methods then I remove comments